cogs/info.py

```diff
@@ -1,17 +1,24 @@
-import time
-import discord
-import psutil
-import os
+import time, discord, psutil, os
 
 from discord.ext import commands
 from utils import default, http
 
 
+class CustomHelp(commands.MinimalHelpCommand):
```

Comment:
Isn't this already controlled by the `utils/data.py`?

Reply:
Sorry about that, I thought it was a built in class. Can I replace the existing help format (in utils/data) with my implementation?

Reply:
I have managed to move this and everything works